### ISSUE TITLE
Not showing in Chrome

### ISSUE THREAD

**Issue by joannesdekoster** [insites/cookieconsent#211]
For some reason the cookieconsent dialog doesn't show on Chrome only (works with Firefox and IE) on [this site](http://www.recrutement-leclerc-montpellier.com/). Do you have any idea what could be wrong?
Thanks for any answer.

**Comment by hidder** [comment]
FYI, for me it is showing correct on chrome
![schermafbeelding 2017-08-04 om 13 06 32](https://user-images.githubusercontent.com/1681167/28966330-cbddc6b8-7915-11e7-86ea-70dafb2b4a07.png)

Maybe some kind of blocker?

**Comment by joannesdekoster** [comment]
I tryed with and without my blocker. Thanks for checking anyway, the problem seems to be only on my side.

**Comment by kissu** [comment]
Hi, I wanted to say that I faced the same issue.
But I found my issue (since cookie-consent was perfectly working on chromium/firefox).
So, if you have some kind of auto accept cookies like [Fuck Cookies](https://chrome.google.com/webstore/detail/fuck-cookies/pngdnkbboiadbedpgcfggkcbpiochkfg?hl=en), even if you open a new incognito window, the text will not display properly, only the "accept" will be.
Or it will be working **only** if you have the incognito window open.

I put it here if any people got the same issue one day. :muscle:
